Doc/library/operator.rst

```diff
@@ -17,9 +17,11 @@
 
 The :mod:`operator` module exports a set of efficient functions corresponding to
 the intrinsic operators of Python.  For example, ``operator.add(x, y)`` is
-equivalent to the expression ``x+y``.  The function names are those used for
-special class methods; variants without leading and trailing ``__`` are also
-provided for convenience.
+equivalent to the expression ``x+y``. Many function names are those used for
+special methods, without the double underscores.  For backward compatibility,
+many of these have a variant with the double underscores kept.  We recommend
+using the dunderless form. Note that ``operator.__add__(x, y)``, for instance,
+being the same as ``x + y``, is not the same as ``x.__add__(y)``.
 
```

Comment:
_Dunderless_ sounds like a made-up term or jargon. I don’t think the documentation even uses _dunder_ anywhere. I would write it as “the variants without the double underscores”.